Updated issue template to guide users to try running RViz with C locale (#1682)
ros-visualization/rviz: .github/ISSUE_TEMPLATE.md

    paste your console output here
    ```
* If source build, which git commit?

> * System locale: e.g. `en_US.UTF-8` (get it by running `echo "$LANG $LC_NUMERIC"` on Linux; before reporting a rendering issue, try running RViz with `LANG=C rviz` and report whether the error is reproducible even in this case)

Reviewer: ```suggestion
* System locale, i.e. the output of `echo "$LANG $LC_NUMERIC"`:
  Before reporting a rendering issue, try running RViz with `LANG=C rviz`!
```
